kinesis.tf

```diff
@@ -1,41 +1,35 @@
 module "kinesis_label" {
-  source     = "git::https://github.com/cloudposse/terraform-null-label.git?ref=tags/0.3.1"
-  namespace  = "${var.namespace}"
-  name       = "${var.name}"
-  stage      = "${var.stage}"
-  delimiter  = "${var.delimiter}"
-  attributes = "${compact(concat(var.attributes, list("kinesis")))}"
-  tags       = "${var.tags}"
-  enabled    = "${var.enabled}"
+  source  = "cloudposse/label/null"
+  version = "0.24.1"
+
+  context    = module.this.context
+  attributes = compact(concat(module.this.attributes, ["kinesis"]))
 }
 
 module "subscription_filter_label" {
-  source     = "git::https://github.com/cloudposse/terraform-null-label.git?ref=tags/0.3.1"
-  namespace  = "${var.namespace}"
-  name       = "${var.name}"
-  stage      = "${var.stage}"
-  delimiter  = "${var.delimiter}"
-  attributes = "${compact(concat(var.attributes, list("filter")))}"
-  tags       = "${var.tags}"
-  enabled    = "${var.enabled}"
+  source  = "cloudposse/label/null"
+  version = "0.24.1"
+
+  context    = module.this.context
+  attributes = compact(concat(module.this.attributes, ["filter"]))
 }
 
 resource "aws_kinesis_stream" "default" {
```

Comment:
There is a variable `encryption_type` which may be passed in.
Seems the scanner want me to change it to 'KMS'
I could change to variable default to 'KMS' instead of 'NONE' or even hard code 'KMS'.

Let me know if I should make the change?

Reply:
Please, add `KMS` as default value for variable

Reply:
Done.